app/src/js/templates.js

```diff
@@ -12,7 +12,7 @@ export const reportHeadingTemplate = function(data) {
 }
 
 export function projectCardTemplate(data) {
-    const baseUrl =  window.location.hostname === "https://dsacms.github.io" ? '/metrics' : ""
+    const baseUrl = window.location.hostname === "dsacms.github.io" ? '/metrics' : ""
```

Comment:
This fixes the path issues @natalialuzuriaga? If so, we should document this somewhere prominent :)

Reply:
Yup, will do!